index.js

```diff
@@ -491,9 +497,9 @@ export default class Carousel extends Component {
     }
 
     render () {
-        const { sliderWidth, itemWidth, containerCustomStyle, contentContainerCustomStyle, enableMomentum } = this.props;
+        const { sliderWidth, itemWidth, containerCustomStyle, contentContainerCustomStyle, enableMomentum, carouselSideMargin } = this.props;
 
-        const containerSideMargin = (sliderWidth - itemWidth) / 2;
+        const containerSideMargin = carouselSideMargin || this.getContainerSideMargin(sliderWidth, itemWidth);
```

Comment:
@skeie Hi there! Unless I'm mistaken, I think you forgot to add the method `getContainerSideMargin()` ;-)